vispy/scene/widgets/widget.py

```diff
@@ -224,21 +244,21 @@ def add_grid(self, *args, **kwargs):
         """
         Create a new Grid and add it as a child widget.
 
-        All arguments are given to add_widget().
+        All arguments are given to Grid().
```

Comment:
This is a change because `add_widget` didn't take any `args` or `kwargs`, so I think they should probably go to `Grid` (and below, `ViewBox`)